uk-ua/javascript-ua.html.markdown

```diff
@@ -3,11 +3,11 @@ language: javascript
 contributors:
     - ["Adam Brenecki", "http://adam.brenecki.id.au"]
     - ["Ariel Krakowski", "http://www.learneroo.com"]
-filename: javascript-ru.js
+filename: javascript-ua.js
 translators:
-  - ["Alexey Gonchar", "http://github.com/finico"]
-  - ["Andre Polykanine", "https://github.com/Oire"]
-lang: ru-ru
+  - ["Ivan Neznayu", "https://github.com/IvanEh"]
```

Comment:
:)) Let him be simply Ivan, probably he'll chime in. Wait, we can mention Ivan here: @IvanEh, how should we mention you in the translators section?)